Alignment/ReferenceTrajectories/plugins/TwoBodyDecayTrajectoryFactory.cc

```diff
@@ -93,15 +93,18 @@ const TrajectoryFactoryBase::ReferenceTrajectoryCollection TwoBodyDecayTrajector
   edm::ESHandle<MagneticField> magneticField;
   setup.get<IdealMagneticFieldRecord>().get(magneticField);
 
+  edm::ESHandle<GlobalTrackingGeometry> trackingGeometry;
+  setup.get<GlobalTrackingGeometryRecord>().get(trackingGeometry);
```

Comment:
Isn't this syntax deprecated?

Reply:
Yes. However I decided I did not want to try modifying this code and all code that uses it to support esConsumes. The goal of this PR was really just to update `PFCand_AssoMap` since it is used in the IBs. The change above will make it easier in the future to migrate this code to esConsumes.